kv/etcd/store.go

```diff
@@ -522,8 +522,13 @@ func newValue(val []byte, ver, rev int64) *value {
 	}
 }
 
-func (c *value) isNewer(other *value) bool {
-	return c.Rev > other.Rev
+func (c *value) IsNewer(other kv.Value) bool {
+	othervalue, ok := other.(*value)
```

Comment:
I'd rather keep it this way in case the user uses 2 different kv stores, say etcd + cassandra or etcd + flipr, and different impls of kv.Value may exist in the runtime, it's probably safer not to just panic